### ISSUE TITLE
Allow keywords in media-feature-value-dollar-variable

### ISSUE THREAD

**Issue by guoyunhe** [kristerkari/stylelint-scss#438]
```scss
// valid
@media screen and (pointer: fine) and (min-width: $breakpoint) {}

// invalid
@media screen and (prefers-color-scheme: dark) and (min-width: 768px) {}
```
In my project, I want all media feature use variables for sizes. But I don't want to define variables for standard keywords like `fine`, `dark`.

**Comment by kristerkari** [comment]
This is now fixed in v3.15.0
https://github.com/kristerkari/stylelint-scss/releases/tag/3.15.0

Thanks @guoyunhe !

**Comment by guoyunhe** [comment]
cheers!
